src/python/WMCore/Storage/Backends/XRDCPImpl.py

```diff
@@ -111,6 +114,9 @@ def createStageOutCommand(self, sourcePFN, targetPFN, options=None, checksums=No
                     for initFile in initFiles:
                         copyCommand += "source %s\n" % initFile
 
+        if args.wma_disablewriterecovery and not args.wma_cerncastor:
```

Comment:
I know I wrote that line, but I think we can remove the "and not args.wma_cerncastor". It's redundant and makes the code harder to read. We only use castor mode from LogCollect anyways where we control the options we set.